spyder/plugins/__init__.py

```diff
@@ -37,6 +37,7 @@
 from spyder.py3compat import configparser, is_text_string
 from spyder.utils import icon_manager as ima
 from spyder.utils.qthelpers import create_action, toggle_actions
+from spyder.plugins.configdialog import PluginConfigPage  # Plugin compatibility

```

Comment:
I didn't spot this, but it doesn't look like it's a problem. It's pretty common to import submodules in `__init__.py`.  See for example this discussion: https://groups.google.com/forum/#!topic/comp.lang.python/HYChxtsrhnw. In that case it's fine.

Now if you want me to remove it because it keeps the code clean or whatever, I can do it !